storagegroup/storagegroup.go

```diff
@@ -206,3 +211,77 @@ func formatCheck(v2 *storagegroup.StorageGroup) error {
 
 	return nil
 }
+
+// ReadFromObject assemble StorageGroup from a regular
+// Object structure. Object must contain unambiguous information
+// about its expiration epoch, otherwise behaviour is undefined.
+//
+// Returns any error appeared during storage group parsing.
+func ReadFromObject(sg *StorageGroup, o objectSDK.Object) error {
+	err := sg.Unmarshal(o.Payload())
+	if err != nil {
+		return fmt.Errorf("could not unmarshal object: %w", err)
+	}
+
+	var expObj uint64
+
+	for _, attr := range o.Attributes() {
+		if attr.Key() == objectV2.SysAttributeExpEpoch {
+			expObj, err = strconv.ParseUint(attr.Value(), 10, 64)
+			if err != nil {
+				return fmt.Errorf("could not get expiration from object: %w", err)
+			}
+
+			break
+		}
+	}
+
+	// Supporting deprecated functionality.
+	// See https://github.com/nspcc-dev/neofs-api/pull/205.
+	if expSG := sg.ExpirationEpoch(); expObj != expSG {
+		return fmt.Errorf(
+			"expiration does not match: from object: %d, from payload: %d",
+			expObj, expSG)
+	}
+
+	return nil
+}
+
+// WriteToObject writes StorageGroup to a regular
+// Object structure. Object must not contain ambiguous
+// information about its expiration epoch.
+func WriteToObject(sg StorageGroup, o *objectSDK.Object) {
+	sgRaw, err := sg.Marshal()
+	if err != nil {
+		// Marshal() does not return errors
+		// in the next API release
+		panic(fmt.Errorf("could not marshal storage group: %w", err))
+	}
+
+	o.SetPayload(sgRaw)
```

Comment:
What about object type? We need to set it in `WriteToObject`, or at least mention about this in docs (which seems much less convenient).

Btw `ReadFromObject` also needs some type documentation.

Reply:
missed that, added object type setting inside `ReadFromObject`, added info to the comment about what exactly fields are set

Reply:
> is that what you meant?

Oh, not exactly. I meant:

- just set type in `WriteToObject` function body, not mention written information in docs (although it's not bad)
- mention in `ReadFromObject` that the function makes sense only for objects with `object.TypeStorageGroup` type (we can )

Reply:
got it. added

Reply:
I think the rule here is to have the package-defined entity as the first argument:
1. This plays nicely with the naming.
2. For example, `oid.ID.ReadFromV2` / `oid.ID.WriteToV2` have the same order for the parameters (when used as 2-argument functions).

Reply:
have thought about that one more time and decided that I agree with @fyrchik: `copy` accepts the same type args so comprising is not that valid

I would:

1. place a `StorageGroup` in the first place
2. make it a method (but we do not do that in the other places of the SDK)

@cthulhu-rider, what do you think?

Reply:
IMO helper functions have nothing to do with methods. Here it is a matter of taste, so I suggest @carpawell make the final decision.

I personally use the rule to follow the variable assignment mnemonic:
```
b := a
copy(b, a)
io.Copy(b, a)
f(b, a)
```
Also it's easy to support variadic input `f(b, a1, a2, a3)`.

Reply:
ok, i keept func and placed SG at the first place